frontend/src/app/main/ui/inspect/attributes/visibility.cljs

```diff
@@ -0,0 +1,62 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; Copyright (c) KALEIDOS INC
+
+(ns app.main.ui.inspect.attributes.visibility
+  (:require-macros [app.main.style :as stl])
+  (:require
+   [app.common.data :as d]
+   [app.common.data.macros :as dm]
+   [app.main.ui.components.copy-button :refer [copy-button*]]
+   [app.main.ui.components.title-bar :refer [inspect-title-bar*]]
+   [app.main.ui.inspect.attributes.common :as cmm]
+   [app.util.code-gen.style-css :as css]
+   [rumext.v2 :as mf]))
+
+(def properties
+  [:opacity
+   :blend-mode
+   :visibility])
+
+(defn has-visibility-props? [shape]
+  (and
+   (not (contains? #{:text :group} (:type shape)))
+   (or (:opacity shape)
+       (:blend-mode shape)
+       (:visibility shape))))
+
+(mf/defc visibility-block
```

Comment:
New components should follow the current convention. It should end with `*` in the name and use `[:>` for create the element on jsx